### PR TITLE
Implemented basic join game page, creating the js file and importing …

### PR DESCRIPTION
**Pull request by IqbalBan**
…and using the component on App.js

### CODE REVIEW

File: my-app/src/joinGamePage.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+function JoinGamePage(){
+    return (
+        
+        <div>
```
Comment (SachJay) [inline my-app/src/joinGamePage.js]:
```suggestion
import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';

function JoinGamePage(){
    return (
        <div>
```

minor spacing and formating

File: my-app/src/components/joinGamePage/joinGamePage.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+function JoinGamePage(){
+    return (
+        
+        <div>
+            <title> Project with the BOIS </title>
+            <div className ="container">
+                <form>
+                    <div className ="form-group">
+                        <label for="text">ROOM CODE</label>
+                        <input type="text" class="form-control" id="inputRoomCode" placeholder="ENTER 4-LETTER CODE"  maxlength="4"  autocapitalize="off" autocorrect="off" autocomplete="off"/>
```
Comment (SachJay) [inline my-app/src/components/joinGamePage/joinGamePage.js:12]: so what are all these params for can u go over them Mr head bootstrap man

**Review by AndCrit — CHANGES_REQUESTED** [review]
IDK if it's just be but it doesn't compile xddd. 
image
Then the local host site looks like this.
image

**Comment by AndCrit** [comment]
So the other thing is I know the button doesn't lead anywhere for now because you have to wait on the ticket "rout people to the correct room based on entered code". Regardless your gonna most prolly put more logic here like a fetch endpoint of the new link or something. So then I think you should add a FIXME just saying that your waiting on that specific ticket before you can move further.
image

**Review by SachJay — APPROVED** [review]
passable

**Review by AndCrit — APPROVED** [review]
OH KAY

**Pull request merged**